message_definitions/v1.0/messages.xml

```diff
@@ -1901,16 +1901,26 @@
     </message>
 
     <message name="STAB_ATTITUDE_INDI" id="216">
-      <field name="angular_accel_p" type="float"/>
-      <field name="angular_accel_q" type="float"/>
-      <field name="angular_accel_r" type="float"/>
-      <field name="angular_accel_ref_p" type="float"/>
-      <field name="angular_accel_ref_q" type="float"/>
-      <field name="angular_accel_ref_r" type="float"/>
-      <field name="g1_p" type="float"/>
+      <field name="body_accel_x" type="float" unit="m/s2"/><!-- setpoint -->
+      <field name="body_accel_y" type="float" unit="m/s2"/>
+      <field name="body_accel_z" type="float" unit="m/s2"/>
+      <field name="angular_rate_p" type="float" unit="rad/s" alt_unit="deg/s" alt_unit_coef="57.29578"/><!-- rate tracking -->
```

Comment:
`alt_unit_coef` is not needed for units described in https://github.com/paparazzi/pprzlink/blob/master/message_definitions/common/units.xml

Reply:
fixed.